build.rs

```diff
@@ -7,37 +7,16 @@ extern crate cc;
 extern crate walkdir;
 
 use std::env;
-use std::path::{Path, PathBuf};
 use std::ffi::{OsStr, OsString};
 use std::fs;
+use std::path::{Path, PathBuf};
 use std::process::Command;
 use std::str;
 use walkdir::WalkDir;
 
-const ENV_VARS: &'static [&'static str] = &[
-    "AR",
-    "AS",
-    "CC",
-    "CFLAGS",
-    "CLANGFLAGS",
-    "CPP",
-    "CPPFLAGS",
-    "CXX",
-    "CXXFLAGS",
-    "MAKE",
-    "MOZ_TOOLS",
-    "MOZJS_FORCE_RERUN",
-    "MOZTOOLS_PATH",
-    "PYTHON",
-    "STLPORT_LIBS",
-];
+const ENV_VARS: &'static [&'static str] = &["AR", "AS", "CC", "CFLAGS", "CLANGFLAGS", "CPP", "CPPFLAGS", "CXX", "CXXFLAGS", "MAKE", "MOZ_TOOLS", "MOZJS_FORCE_RERUN", "MOZTOOLS_PATH", "PYTHON", "STLPORT_LIBS"];
```

Comment:
I don't understand what happened in this commit. Can we revert it?

Reply:
I ran `cargo fmt`

Reply:
It's due to https://github.com/servo/mozjs/blob/master/rustfmt.toml#L2. Should we do something like https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#configuration-file-resolution and make sure it is only applied on jsapi files?

Reply:
Yeah, that would be sensible.